### ISSUE TITLE
Add text to "Leadership in Data Science"

### ISSUE THREAD

**Issue by HeidiSeibold** [alan-turing-institute/the-turing-way#1224]
### Summary

The [Leadership in Data Science](https://the-turing-way.netlify.app/collaboration/leadership.html) currently looks a bit sad with only headers.

![image](https://user-images.githubusercontent.com/14146757/84877852-4dc53780-b089-11ea-9d8c-9840e24a72b0.png)

If we'd add at least a bit of text, it might be a big improvement.

### What needs to be done?

- [ ] Add some intro text what this chapter will be about

### Who can help?

* Anyone who know (or can find out) what this chapter is supposed to be about.

---

### Updates

Nothing yet

**Comment by malvikasharan** [comment]
@nadiasoliman I am tagging you as I know that you have some stuff in progress for this section. Will you be able to help with this?

**Comment by nadiasoliman** [comment]
Absolutely!  @malvikasharan I am sorry it's taking me so long to start adding.  I will do so by 10 July - I need to plan with hard deadlines as things are so busy at the moment and please do hold me to account

**Comment by malvikasharan** [comment]
@lauracion is drafting the chapter. 

We will welcome a specific case study or story on different leadership roles in Data Research. For example, @sgibson91 has contributed a personal story from her experience leading a project at a hackthon event: https://the-turing-way.netlify.app/collaboration/leadership/leadership-stories.html

**Comment by HeidiSeibold** [comment]
:clap: wow I am excited to learn from @lauracion! Big fan! :nerd_face:

**Comment by lauracion** [comment]
Thank you, Heidi. Admiration is mutual 😄 I am hoping to soon have an initial draft to share with you all. Although slow, steady progress is happening.

I am super excited because @luucamay, from El Alto (Bolivia), accepted my invitation to contribute a personal story from her experience leading a project in open source🎉

**Comment by HeidiSeibold** [comment]
Looking at the chapter, I think this issue can be closed 😉
